protobuf/scalapb/scalapb.proto

```diff
@@ -291,6 +291,18 @@ message EnumOptions {
   // must be present.
   optional string type = 3;
 
+  // Custom annotations to add to the generated case object.
+  repeated string annotations_for_trait = 4;
+
+  // Custom annotations to add to the generated case object.
```

Comment:
I think it should be "all generated case objects"? Is it including or excluding the Unrecognized - let's specify.